### PR TITLE
🌱 Trying to fix doc generation

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

File: .github/workflows/docs-gen-and-push.yaml [hunk 1]
```diff
@@ -125,7 +125,10 @@ jobs:
 
       # copy back older versions to the current docs
       - name: Copy everything from archive to docs
-        run: rsync -av docs/v* public
+        run: |
+          if ls docs/v* >/dev/null; then
+            rsync -av docs/v* public
+          fi
```
Comment (avinal) [inline .github/workflows/docs-gen-and-push.yaml:131]: Should we add an else branch as well, printing "No such folder"? will be easier to track in logs. (probably an overkill)

**Review by avinal — APPROVED** [review]
Looks Good to me

**Comment by ncdc** [comment]
Flake #2713, which should be resolved now. Hopefully another retest will get past that.
/retest

**Pull request merged**